tests/integration-tests/tests/networking/test_networking.py

```diff
@@ -0,0 +1,188 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Let's move test functions at the top of the file and use the _ prefix for all local functions